scripts/generate_pinval/generate_pinval.py

```
@@ -323,7 +324,7 @@ def convert_dtypes(df: pd.DataFrame) -> pd.DataFrame:
 
     for col, dtype in dtype_conversions.items():
         if col in df.columns:
-            df[col] = pd.to_numeric(df[col], errors="coerce").astype(dtype)
+            df[col] = pd.to_numeric(df[col], errors="raise").astype(dtype)
```

I figured this should fail loudly instead of coercing